Modules/Loadable/VolumeRendering/MRML/vtkMRMLShaderPropertyNode.cxx

```diff
@@ -108,22 +107,16 @@ void vtkMRMLShaderPropertyNode::Copy(vtkMRMLNode *anode)
 
   this->Superclass::Copy(anode);
 
-  this->CopyParameterSet(anode);
```

Comment:
```
disabledModify = this->StartModify();
``` 

should be removed as well. 

Consider rebasing the topic. This should allow the CI to be green.